komga-webui/src/locales/en.json

```diff
@@ -164,12 +164,12 @@
   },
   "browse_readlist": {
     "edit_elements": "Edit elements",
-    "edit_readlist": "Edit read list"
```

Comment:
This is the same unreferenced awkwardness.

Reply:
Read list echoes to play list. You don't see playing list on Spotify though?

Reply:
While I don't use Spotify, my point is that it isn't "play list" either. (Playlist.)
"Edit" (past tense" read → list, is how I grasp this at first.
I want to avoid "(A) edit read (B) list" where "the" is either A or B.

Reply:
playlist is not just in spotify, but any media application…

i agress that it is `playlist`, not `play list`, as such i am happy to change all the `read list` occurences to `readlist`, but not to `reading list`.

Reply:
"readlist" isn't a word, but https://dictionary.cambridge.org/dictionary/english/reading-list is. 
Consequently, this is how you get "The bug reports".

Let's say you even go so far as to get it actively wrong in translation, and now it really says ~"Edit the list that has been read", instead of just opening that door.

Reply:
Fair enough, let's go for "reading list" everywhere then.